src/components/templates/settingsLayout.tsx

```diff
@@ -28,12 +40,16 @@ export const SettingsLayout = () => {
 					<Sidebar />
 
 					<div className="flex flex-1 flex-col">
-						<TitleTopbar title={tSettings("title")} />
+						<TitleTopbar title={`${tSettings("userSettings")}: ${subPageTitle}`} />
```

Comment:
On "/settings/client-configuration" without the flag you get:

![image](https://github.com/user-attachments/assets/922c69fa-5db9-4a8b-a57b-2bb766577535)

Reply:
fixed